components/search_results/search_results.jsx

```diff
@@ -337,7 +337,6 @@ export default class SearchResults extends React.PureComponent {
                         user={profile}
                         term={searchTerms}
                         isMentionSearch={this.props.isMentionSearch}
-                        isFlaggedSearch={this.props.isFlaggedPosts}
```

Comment:
This is not related to this PR but good to have as part of code cleanup. Would be nice to remove this as well https://github.com/mattermost/mattermost-webapp/blob/master/components/search_results_item.jsx#L34

Reply:
I have that as part of PR :)

Reply:
Cool, didn't see that below :)